### ISSUE TITLE
The latest pixel 4 module failed to install

### ISSUE THREAD

**Issue by vl3550** [Pixel-Props/coral#1]
The latest pixel 4 module installation failed. Please fix

**Comment by 0x11DFE** [comment]
Could you be more precise?
Have you tried to flash it using Magisk?

**Comment by vl3550** [comment]
> Could you be more precise?
> Have you tried to flash it using Magisk?

Yes, I used magisk to install him, but he said the installation failed

**Comment by vl3550** [comment]
[magisk_install_log_2021-06-24T06_50_39Z.txt](https://github.com/Pixel-Props/coral/files/6706310/magisk_install_log_2021-06-24T06_50_39Z.txt)

**Comment by 0x11DFE** [comment]
> [magisk_install_log_2021-06-24T06_50_39Z.txt](https://github.com/Pixel-Props/coral/files/6706310/magisk_install_log_2021-06-24T06_50_39Z.txt)

From what i understand it failed to `grep_prop` have you tried to flash the latest Busybox?

**Comment by vl3550** [comment]
> > [magisk_install_log_2021-06-24T06_50_39Z.txt](https://github.com/Pixel-Props/coral/files/6706310/magisk_install_log_2021-06-24T06_50_39Z.txt)
> 
> From what i understand it failed to `grep_prop` have you tried to flash the latest Busybo

> > [magisk_install_log_2021-06-24T06_50_39Z.txt](https://github.com/Pixel-Props/coral/files/6706310/magisk_install_log_2021-06-24T06_50_39Z.txt)
> 
> 据我所知，`grep_prop`您尝试刷新最新的 Busybox失败？

I did not install busybox

**Comment by vl3550** [comment]
> > [magisk_install_log_2021-06-24T06_50_39Z.txt](https://github.com/Pixel-Props/coral/files/6706310/magisk_install_log_2021-06-24T06_50_39Z.txt)
> 
> From what i understand it failed to `grep_prop` have you tried to flash the latest Busybox?

Do you mean that you need to install the busybox module before installing the pixel4 module?

**Comment by 0x11DFE** [comment]
> > > [magisk_install_log_2021-06-24T06_50_39Z.txt](https://github.com/Pixel-Props/coral/files/6706310/magisk_install_log_2021-06-24T06_50_39Z.txt)
> > 
> > 
> > From what i understand it failed to `grep_prop` have you tried to flash the latest Busybox?
> 
> Do you mean that you need to install the busybox module before installing the pixel4 module?

It is recommended, I always have it installed by default.

**Comment by vl3550** [comment]
> > > > [magisk_install_log_2021-06-24T06_50_39Z.txt](https://github.com/Pixel-Props/coral/files/6706310/magisk_install_log_2021-06-24T06_50_39Z.txt)
> > > 
> > > 
> > > From what i understand it failed to `grep_prop` have you tried to flash the latest Busybox?
> > 
> > 
> > Do you mean that you need to install the busybox module before installing the pixel4 module?
> 
> It is recommended, I always have it installed by default.

I just tried to install the busybox module and restarted to install the pixel 4xl module again. He still said the installation failed.

**Comment by 0x11DFE** [comment]
> > > > > [magisk_install_log_2021-06-24T06_50_39Z.txt](https://github.com/Pixel-Props/coral/files/6706310/magisk_install_log_2021-06-24T06_50_39Z.txt)
> > > > 
> > > > 
> > > > From what i understand it failed to `grep_prop` have you tried to flash the latest Busybox?
> > > 
> > > 
> > > Do you mean that you need to install the busybox module before installing the pixel4 module?
> > 
> > 
> > It is recommended, I always have it installed by default.
> 
> I just tried to install the busybox module and restarted to install the pixel 4xl module again. He still said the installation failed.

Does the error still occurs on a different module? e.g [Pixel 5 (Redfin)](https://github.com/Pixel-Props/redfin)

**Comment by vl3550** [comment]
> > > > > > [magisk_install_log_2021-06-24T06_50_39Z.txt](https://github.com/Pixel-Props/coral/files/6706310/magisk_install_log_2021-06-24T06_50_39Z.txt)
> > > > > 
> > > > > 
> > > > > 据我所知，`grep_prop`您尝试刷新最新的 Busybox失败？
> > > > 
> > > > 
> > > > 你的意思是在安装pixel4模块之前需要先安装busybox模块吗？
> > > 
> > > 
> > > 推荐使用，我总是默认安装它。
> > 
> > 
> > 我只是尝试安装busybox模块并重新启动以再次安装像素4xl模块。他还是说安装失败。
> 
> 错误是否仍然发生在不同的模块上？例如[Pixel 5 (Redfin)](https://github.com/Pixel-Props/redfin)



> > > > > > [magisk_install_log_2021-06-24T06_50_39Z.txt](https://github.com/Pixel-Props/coral/files/6706310/magisk_install_log_2021-06-24T06_50_39Z.txt)
> > > > > 
> > > > > 
> > > > > From what i understand it failed to `grep_prop` have you tried to flash the latest Busybox?
> > > > 
> > > > 
> > > > Do you mean that you need to install the busybox module before installing the pixel4 module?
> > > 
> > > 
> > > It is recommended, I always have it installed by default.
> > 
> > 
> > I just tried to install the busybox module and restarted to install the pixel 4xl module again. He still said the installation failed.
> 
> Does the error still occurs on a different module? e.g [Pixel 5 (Redfin)](https://github.com/Pixel-Props/redfin)

The pixel 5 module is installed normally, it can be installed successfully and runs perfectly!

**Comment by 0x11DFE** [comment]
> The pixel 5 module is installed normally, it can be installed successfully and runs perfectly!

Then use the PIxel 5 module until i drag a new release which will be probably next month.

**Comment by vl3550** [comment]
> > 像素5模块安装正常，可以安装成功，完美运行！
> 
> 然后使用 PIxel 5 模块，直到我拖出一个新版本，这可能是下个月。



> > The pixel 5 module is installed normally, it can be installed successfully and runs perfectly!
> 
> Then use the PIxel 5 module until i drag a new release which will be probably next month.

OK OK

**Comment by 0x11DFE** [comment]
Does the same problem still occurs on the latest release [coral-user 11 RQ3A.210705.001 7380771 release-keys](https://github.com/Pixel-Props/coral/releases/tag/210705) ?

**Comment by vl3550** [comment]
> Does the same problem still occurs on the latest release [coral-user 11 RQ3A.210705.001 7380771 release-keys](https://github.com/Pixel-Props/coral/releases/tag/210705) ?

Yes, he still has the same problem. Installation failed!

**Comment by 0x11DFE** [comment]
> > Does the same problem still occurs on the latest release [coral-user 11 RQ3A.210705.001 7380771 release-keys](https://github.com/Pixel-Props/coral/releases/tag/210705) ?
> 
> Yes, he still has the same problem. Installation failed!

Is the issue fixed?
